tests/bluetooth/bsim_bt/bsim_test_notify/src/gatt_server_test.c

```diff
@@ -163,6 +163,40 @@ static inline void long_notify(void)
 	} while (err);
 }
 
+static inline void multiple_notify(void)
+{
+	static size_t short_length = CHRC_SIZE;
+	static struct bt_gatt_notify_params params[] = {
+		{
+			.attr = &attr_test_svc[4],
+			.data = long_chrc_data,
+			.len = LONG_CHRC_SIZE,
+			.func = notification_sent,
+			.user_data = &short_length,
+			.uuid = NULL,
+		},
+		{
+			.attr = &attr_test_svc[1],
+			.data = chrc_data,
+			.len = CHRC_SIZE,
+			.func = notification_sent,
+			.user_data = &short_length,
+			.uuid = NULL,
+		},
+	};
+	int err;
+
+	do {
+		err = bt_gatt_notify_multiple(g_conn, ARRAY_SIZE(params), params);
+
+		if (err == -ENOMEM) {
+			k_sleep(K_MSEC(10));
+		} else if (err) {
```

Comment:
Don't we also get `ENOMEM` if the TX buffer size isn't large enough? In that case we'll just try indefinitely.

Reply:
@Thalley I see it only returns ENOMEM when calling bt_att_create_pdu failed.